variables.tf

```diff
@@ -196,6 +196,9 @@ variable "s3_replication_rules" {
   #       owner = string
   #     })
   #     account_id                 = string
+  #     metrics                    = object({
+  #       status = Enabled
```

Comment:
Value on the RHS — even when commented out — should be the type, not the default 

```suggestion
  #       status = string
```